addons/mass_mailing_event/models/event_event.py

```diff
@@ -16,7 +16,11 @@ def action_mass_mailing_attendees(self):
             'target': 'current',
             'context': {
                 'default_mailing_model_id': self.env.ref('event.model_event_registration').id,
-                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', 'not in', ['cancel', 'draft'])])
+                # keys 'mailing_model_id' and 'mailing_domain' are needed because 'default_' keys
+                # are removed from context when processing a compute method with compute_sudo
+                # flag enabled
+                'mailing_model_id': self.env.ref('event.model_event_registration').id,
+                'mailing_domain': repr([('event_id', 'in', self.ids), ('state', 'not in', ['cancel', 'draft'])]),
```

Comment:
Why a repr(list) ?

Reply:
was like that before, comes from https://github.com/odoo/odoo/pull/41877/commits/9600f2c5a1acf367ad8091867a58cd8ea7f9fbf8#r358249330
It worked, so no further reflexion from my part :sweat_smile: